app/Console/Commands/RefreshCampaignCounts.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Campaign;
+use Illuminate\Console\Command;
+
+class RefreshCampaignCounts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'campaign:refresh-counts';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Refresh post counts for all campaigns.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        foreach (Campaign::cursor() as $campaign) {
+            $this->line('Refreshing cached post counts for ' . $campaign->id . '.');
+            $campaign->refreshCounts();
```

Comment:
Aren't we gonna have to run this pretty often for it to not be confusing for admins? Like if you review all the posts, then come back to the campaign index, won't it be confusing if the inbox number hasn't changed?

Reply:
We [run this on-demand with a model event](https://github.com/DoSomething/rogue/pull/947/files#diff-01de341882d0c02d6377ad12e035b315R57) so it'll always be up-to-date after reviewing a post! I'm thinking we'll run the scheduled job every 5-10 minutes or so, so that won't be too out-of-date either.